copier/copier.go

```diff
@@ -1179,10 +1180,10 @@ func copierHandlerGet(bulkWriter io.Writer, req request, pm *fileutils.PatternMa
 				// we don't expand any of the contents that are archives
 				options := req.GetOptions
 				options.ExpandArchives = false
-				walkfn := func(path string, info os.FileInfo, err error) error {
+				walkfn := func(path string, d fs.DirEntry, err error) error {
```

Comment:
I hate single char names for a var, especially if they are not referenced withing a few lines of code.  It's hard to search for by eye, and even with using search.  There are a number of 'd' chars between the definition and it's uses below.  Can you go to at least 3 chars?